src/components/Text/Text.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react';
+import { stylesheet, style } from 'typestyle';
+import { ThemeConsumer, Theme } from '../Theme/Theme';
+/**
+ * Text component to style your `Text` effortlessly.
+ *
+ * **Usage**
+ * ```jsx
+ * <Text>Hello</Text>
+ * ```
+ */
+export const Text: React.FunctionComponent<TextProps> = (props) => {
+  const display = (theme: Theme) => {
+    let fontWeight: number = 400;
+    let fontStyle = 'normal';
+    const fontFamily = props.fontFamily;
+    const fontSize = `${props.fontSize}px`;
+    const color = props.color;
+    let padding: string;
+    let margin: string;
+    let textDecoration: string;
```

Comment:
```ts
let  textDecoration = '';
if (props.underline) {
  textDecoration += 'underline';
}
if (props.strikeThrough) {
  textDecoration += ' line-through'
}
```

using else looks bad.. try to implement this change else where. See where we can fit it.